doc/api/tls.md

```diff
@@ -842,6 +842,14 @@ See
 [OpenSSL](https://www.openssl.org/docs/man1.1.1/man3/SSL_CIPHER_get_name.html)
 for more information.
 
+### tlsSocket.getSharedSigalgs()
+* Returns: {Array} List of signature algorithms shared between the server and
+the client in the order of decreasing preference.
+
+See
+[OpenSSL](https://www.openssl.org/docs/man1.1.1/man3/SSL_get_shared_sigalgs.html)
```

Comment:
please do ```s/OpenSSL/`SSL_get_shared_sigalgs`/```, like the link to SSL_get_version

Reply:
The same for ciphers just above probably?

Reply:
I wouldn't ask you to change up code you didn't write and that's unrelated to this PR, but yes, that other link could be improved, too.